### PR TITLE
Bundle related methods

### PR DESCRIPTION
**Pull request by emilyploszaj**

### CODE REVIEW

File: mappings/net/minecraft/item/Item.mapping [hunk 1]
```diff
@@ -21,6 +21,12 @@ CLASS net/minecraft/class_1792 net/minecraft/item/Item
 	METHOD method_24357 damage (Lnet/minecraft/class_1282;)Z
 		ARG 1 source
 	METHOD method_24358 isFireproof ()Z
+	METHOD method_31565 onStackClicked (Lnet/minecraft/class_1799;Lnet/minecraft/class_1799;Lnet/minecraft/class_5536;Lnet/minecraft/class_1661;)Z
+	METHOD method_31566 onClicked (Lnet/minecraft/class_1799;Lnet/minecraft/class_1799;Lnet/minecraft/class_5536;Lnet/minecraft/class_1661;)Z
+	METHOD method_31567 showItemBar (Lnet/minecraft/class_1799;)Z
```
Comment (Gegy) [inline mappings/net/minecraft/item/Item.mapping]: `hasItemBar` / `showsItemBar`?

Comment (haykam821) [reply mappings/net/minecraft/item/Item.mapping]:
```suggestion
	METHOD method_31567 isItemBarVisible (Lnet/minecraft/class_1799;)Z
```

File: mappings/net/minecraft/item/Item.mapping [hunk 2]
```diff
@@ -21,6 +21,12 @@ CLASS net/minecraft/class_1792 net/minecraft/item/Item
...
+	METHOD method_31568 hasStoredInventory ()Z
+	METHOD method_31569 getItemBarIncrement (Lnet/minecraft/class_1799;)I
```
Comment (haykam821) [inline mappings/net/minecraft/item/Item.mapping]:
```suggestion
	METHOD method_31569 getItemBarStep (Lnet/minecraft/class_1799;)I
```

**Review by haykam821 — CHANGES_REQUESTED** [review]
Perhaps it would be better to simplify to just 'bar'?

File: mappings/net/minecraft/util/ClickType.mapping [hunk 3]
```diff
@@ -0,0 +1,3 @@
+CLASS net/minecraft/class_5536 net/minecraft/util/ClickType
+	FIELD field_27013 LEFT Lnet/minecraft/class_5536;
+	FIELD field_27014 RIGHT Lnet/minecraft/class_5536;
```
Comment (YanisBft) [inline mappings/net/minecraft/util/ClickType.mapping:3]: What did you not use automapped names?

Comment (emilyploszaj) [reply mappings/net/minecraft/util/ClickType.mapping:3]: primary and secondary seemed less meaningful than left and right when referring to a mouse

**Comment by emilyploszaj** [comment]
> 
> 
> Perhaps it would be better to simplify to just 'bar'?

This was my initial thought but it just seemed better in context to specify it as an item bar

**Review by jaskarth — APPROVED** [review]

**Review by OroArmor — APPROVED** [review]

File: mappings/net/minecraft/item/ItemStack.mapping [hunk 4]
```diff
@@ -38,6 +38,11 @@ CLASS net/minecraft/class_1799 net/minecraft/item/ItemStack
 	METHOD method_30267 isSectionHidden (ILnet/minecraft/class_1799$class_5422;)Z
 		ARG 0 flags
 	METHOD method_30268 addHideFlag (Lnet/minecraft/class_1799$class_5422;)V
+	METHOD method_31575 onStackClicked (Lnet/minecraft/class_1799;Lnet/minecraft/class_5536;Lnet/minecraft/class_1661;)Z
+	METHOD method_31576 onClicked (Lnet/minecraft/class_1799;Lnet/minecraft/class_5536;Lnet/minecraft/class_1661;)Z
+	METHOD method_31578 showItemBar ()Z
```
Comment (Gegy) [inline mappings/net/minecraft/item/ItemStack.mapping]: These mappings should probably match the updated ones in `Item`

Comment (emilyploszaj) [reply mappings/net/minecraft/item/ItemStack.mapping]: Matched to the mappings in `Item`

**Review by YanisBft — APPROVED** [review]

**Review by Gegy — APPROVED** [review]

**Pull request merged**